### DISCUSSION TITLE
Erro ao executar debug

### DISCUSSION THREAD

**Discussion by luistring8** [balta-io/forum#25] [Tira Dúvidas] [answered]
Curso Fundamentos do ASP.NET 6
Módulo CRUD

Não consigo executar o debug, tampouco rodar local via postman

<img width="593" alt="Captura de Tela 2022-09-25 às 11 22 56 AM" src="https://user-images.githubusercontent.com/68756072/192151422-9d0572bb-b278-44da-a115-bb82bc91ae97.png">

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @luistring8, como vai?

Este erro diz que já existe uma aplicação rodando na porta `7138` da sua máquina.
Tenta alterar a porta de execução no arquivo `Properties/` e/ou reiniciar sua máquina.
